Formula/osinfo-db.rb

```diff
@@ -0,0 +1,17 @@
+class OsinfoDb < Formula
+  desc "Osinfo database of operating systems for virtualization provisioning tools"
+  homepage "https://libosinfo.org/"
+  url "https://releases.pagure.org/libosinfo/osinfo-db-20210426.tar.xz"
+  sha256 "66c93b4a5b319b2ff7d40360d9e94486a9a45f2c0dd1c30d4b31de4f223070d0"
+  license "GPL-2.0-or-later"
+
+  depends_on "osinfo-db-tools" => :build
+
+  def install
+    system "osinfo-db-import", "--dir=#{share}/osinfo", cached_download
+  end
+
+  test do
+    system "osinfo-db-validate", "--system"
```

Comment:
Same comment regarding the other test.

Reply:
This command doesn't output anything unless it fails, so there's no output to check. There is for the `osinfo-db-tools` test, though. I'm adding an output check now.